config/manifests/kustomization.yaml

```diff
@@ -9,6 +9,6 @@ resources:
 images:
 - name: controller
   newName: quay.io/submariner/submariner-operator
-  newTag: 0.11.0
+  newTag: devel-efc6ef9e
```

Comment:
Is it correct to reference devel?

Reply:
No, and I think this is contributing to the failures of the bundle test jobs; the published bundles now store the hash of the operator repository, rather than the relevant tag.